optimum/intel/openvino/modeling_open_clip.py

```diff
@@ -0,0 +1,733 @@
+        # This attribute is needed to keep one reference on the temporary directory, since garbage collecting
+        # would end-up removing the directory containing the underlying OpenVINO model
+        cls._model_save_dir_tempdirectory_instance = save_dir
+
+        # If load_in_8bit and quantization_config not specified then ov_config is set to None and will be set by default in convert depending on the model size
+        if load_in_8bit is None and not quantization_config:
+            ov_config = None
+        else:
+            ov_config = OVConfig(dtype="fp32")
+
+        def fn_get_submodels(model):
+            return {"model_text": model.text}
+
+        custom_export_configs = {
+            "model_text": {},
+        }
+
+        main_export(
+            model_name_or_path=model_id,
+            output=save_dir_path,
+            task=task or "zero-shot-image-classification",
+            subfolder=subfolder,
+            revision=revision,
+            cache_dir=cache_dir,
+            token=token,
+            local_files_only=local_files_only,
+            force_download=force_download,
+            trust_remote_code=trust_remote_code,
+            ov_config=ov_config,
+            library_name="open_clip",
+            framework="pt",
+            fn_get_submodels=fn_get_submodels,
+            custom_architecture=True,
+            custom_export_configs=custom_export_configs,
+        )
+
+        config.save_pretrained(save_dir_path)
+        return cls._from_pretrained(
+            model_id=save_dir_path,
+            config=config,
+            load_in_8bit=load_in_8bit,
+            quantization_config=quantization_config,
+            file_name=cls.OV_XML_FILE_NAME,
+            **kwargs,
+        )
+
+    @classmethod
+    def _from_pretrained(
+        cls,
+        model_id: Union[str, Path],
+        config: PretrainedConfig,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        token: Optional[Union[bool, str]] = None,
+        revision: Optional[str] = None,
+        force_download: bool = False,
+        cache_dir: str = HUGGINGFACE_HUB_CACHE,
+        file_name: Optional[str] = "openvino_model_text.xml",
+        subfolder: str = "",
+        from_onnx: bool = False,
+        local_files_only: bool = False,
+        load_in_8bit: bool = False,
+        quantization_config: Union[OVWeightQuantizationConfig, Dict] = None,
+        **kwargs,
+    ):
+        return super()._from_pretrained(
+            model_id=model_id,
+            config=config,
+            use_auth_token=use_auth_token,
+            token=token,
+            revision=revision,
+            force_download=force_download,
+            cache_dir=cache_dir,
+            file_name=file_name,
+            subfolder=subfolder,
+            from_onnx=from_onnx,
+            local_files_only=local_files_only,
+            load_in_8bit=load_in_8bit,
+            quantization_config=quantization_config,
+            **kwargs,
+        )
```

Comment:
why not remove 
```suggestion
```

Reply:
I added it to setup file name for text/vision model `file_name: Optional[str] = "openvino_model_text.xml",` , without this we will look for `openvino_model.xml`

Reply:
what about using `cls.OV_XML_FILE_NAME` in https://github.com/huggingface/optimum-intel/blob/ba74307dfc7f69c6281e811e02ee0203a6fb8355/optimum/intel/openvino/modeling_base.py#L325 directly, maybe renaming to something like `cls._ov_xml_file` or `cls._default_xml_file`